packages/website/components/accordionblock/accordionblock.js

```diff
@@ -0,0 +1,33 @@
+// ===================================================================== Imports
```

Comment:
whao extreme commenting

Reply:
Is there a preferred comment format to delineate sections? @olizilla